ai_transform/utils/document_list.py

```diff
@@ -45,6 +45,19 @@ def _flatten_list(self, list_of_lists):
         flat_list = itertools.chain(*list_of_lists)
         return list(flat_list)
 
+    def set_chunk_values(
```

Comment:
You need to also modify behavior in select_fields so that we get all the values in a chunk field